src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -293,4 +303,41 @@ private static WebACAuthorization createAuthorizationFromMap(final Map<String, L
             return Optional.empty();
         }
     }
+
+    public List<WebACAuthorization> getDefaultAuthorizations() {
+        final Map<String, List<String>> aclTriples = new HashMap<>();
+        final List<WebACAuthorization> authorizations = new ArrayList<>();
+
+        getDefaultAcl().listStatements().forEachRemaining(x -> {
+            if (isAclPredicate.test(x.getPredicate())) {
+                final Triple t = x.asTriple();
+                aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
+                if (t.getObject().isURI()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(t.getObject().getURI());
+                } else if (t.getObject().isLiteral()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(
+                        t.getObject().getLiteralValue().toString());
+                }
+            }
+        });
+
+        authorizations.add(createAuthorizationFromMap(aclTriples));
+
+        return authorizations;
+    }
+
+    private static Model getDefaultAcl() {
+        final String rootAcl = System.getProperty(ROOT_ACL_PROPERTY);
+
+        if (rootAcl != null && new File(rootAcl).isFile()) {
+            LOGGER.debug("Getting Root ACL from file: {}", rootAcl);
+            return RDFDataMgr.loadModel(rootAcl);
+        } else {
+            LOGGER.debug("Getting Root ACL from classpath: {}", ROOT_ACL_LOCATION);
+            final Model model = createDefaultModel();
+            final InputStream is = WebACRolesProvider.class.getResourceAsStream(ROOT_ACL_LOCATION);
+            RDFDataMgr.read(model, is, Lang.TTL);
+            return model;
```

Comment:
I don't think so (at least I tried this and couldn't get it to work). The jena [api docs](https://jena.apache.org/documentation/javadoc/arq/org/apache/jena/riot/RDFDataMgr.html#loadModel-java.lang.String-) show `loadModel` with a signature that accepts a URI, but the Riot file loader seems only to handle files or URLs -- not `classpath` references.